### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by dibyaranjanlf**
Uploaded images to be added to topics.
Signed-off by: dibyaranjan <dibyaranjan@proximabiz.com>

### CODE REVIEW

**Comment by dibyaranjanlf** [comment]
Updated Corporate Contributor user flow.
Signed-off by: dibyaranjan <dibyaranjan@proximabiz.com>

File: contributors/contribute-to-a-gerrit-project.md [hunk 1]
```diff
@@ -61,43 +63,100 @@ You are redirected to Gerrit. Wait a few seconds for the CLA status to update.
...
+
+1. Read the Confirmation of Association statement and select the checkbox.
 
    ![Gerrit Confirmation of Association](../.gitbook/assets/cla-gerrit-confirmation-of-association.png)
 
-4. Click **CONTINUE**.
+2. Click **CONTINUE**.
 
-   A dialog appears and informs you: You are done!
+   A dialog appears and informs yoy: You are done!
```
Comment (WillsonHG) [inline contributors/contribute-to-a-gerrit-project.md]: "you" is misspelled

File: contributors/contribute-to-a-gerrit-project.md [hunk 2]
```diff
@@ -61,43 +63,100 @@ You are redirected to Gerrit. Wait a few seconds for the CLA status to update.
...
+
+2. Click the **Please review the agreement link** and then click the message link that appears:
+
+   ![Gerrit Sign ICLA Link](../.gitbook/assets/cla-gerrit-icla-proceed-to-sign-cla.png)
+   
+3. Sign in to EasyCLA if you are prompted.
+   
+4. Select **Company**.
+
+   To contribute to this project, you must be authorized under a signed Contributor License Agreement. You are contributing on behalf of    your work for a company.
```
Comment (WillsonHG) [inline contributors/contribute-to-a-gerrit-project.md]: Not sure about this big space here between "behalf of" and "your".

File: contributors/contribute-to-a-gerrit-project.md [hunk 3]
```diff
@@ -61,43 +63,100 @@ You are redirected to Gerrit. Wait a few seconds for the CLA status to update.
...
+   A dialog appears and informs yoy: You are done!
+   
+   ![You are done](../.gitbook/assets/cla-github-you-are-done%20%281%29.png)
 
-5. Click **RETURN TO REPO**.
+3. Click **RETURN TO REPO**.
 
    You are redirected to Gerrit. Wait a few seconds for the CLA status to update or refresh the page.
+   
+## If a **Company has not sigend CCLA** window appears
```
Comment (WillsonHG) [inline contributors/contribute-to-a-gerrit-project.md]: "sigend"

File: contributors/contribute-to-a-gerrit-project.md [hunk 4]
```diff
@@ -61,43 +63,100 @@ You are redirected to Gerrit. Wait a few seconds for the CLA status to update.
...
+
+This window apperas if your comapny has not signed a Corporate CLA with Linux Foundation.
```
Comment (WillsonHG) [inline contributors/contribute-to-a-gerrit-project.md]: "This window appears if your company has not signed a Corporate CLA for the project."

File: contributors/contribute-to-a-gerrit-project.md [hunk 5]
```diff
@@ -61,43 +63,100 @@ You are redirected to Gerrit. Wait a few seconds for the CLA status to update.
...
+
+To send an email notification to your company's CLA manager to sign Corporate CLA:
+
+1. Select your email address, that you want your comapny manager to whitelist while signing Corporate CLA, from the **Emial to Authorize** drop-down field.
```
Comment (WillsonHG) [inline contributors/contribute-to-a-gerrit-project.md]: "Select your email address from the **Email to Authorize** drop-down field. This is the email address that you want your company manager to whitelist while signing the Corporate CLA."

File: contributors/contribute-to-a-gerrit-project.md [hunk 6]
```diff
@@ -61,43 +63,100 @@ You are redirected to Gerrit. Wait a few seconds for the CLA status to update.
...
+
+2. Click **SEND**.
+
+  A message shows that your email is successfully sent.
+  
+  ![](/.gitbook/assets/email-to-whitelist.png)
+
+## If You are not whitelisted
+
+This window apperas if your comapny has not whitelisted you under their signed Corporate CLA.
```
Comment (WillsonHG) [inline contributors/contribute-to-a-gerrit-project.md]: "appears" and "company"

File: contributors/contribute-to-a-gerrit-project.md [hunk 7]
```diff
@@ -61,43 +63,100 @@ You are redirected to Gerrit. Wait a few seconds for the CLA status to update.
...
+
+To send a request to your company's CLA manager to be whitelisted:
+
+1. Click **CONTACT**.
+
+   ![](/.gitbook/assets/request-to-be-whitelisted.png)
+
+    A **Request Access** window appears.
+  
+2. Select your email address, that you want your comapny manager to whitelist while signing Corporate CLA, from the **Emial to Authorize** drop-down field, and type a message.
```
Comment (WillsonHG) [inline contributors/contribute-to-a-gerrit-project.md]: "Select your email address from the Email to Authorize drop-down field. This is the email address that you want your company manager to whitelist while signing the Corporate CLA."

File: contributors/contribute-to-a-gerrit-project.md [hunk 8]
```diff
@@ -61,43 +63,100 @@ You are redirected to Gerrit. Wait a few seconds for the CLA status to update.
...
-7. Click **CONTINUE**.
+1. Click **COMPANY NOT IN LIST? CLICK HERE**.
 
-   A dialog appears and informs you: You are done!
+   The **Verify Your Permission of Access** dialog appears.
+  
+2. Click an answer: Are You a CLA Manager?
 
-8. Click **RETURN TO REPO**.
+   **YES**— You will be reditected to [corporate.lfcla.com](https://corporate.lfcla.com/#/companies) to link your company to signed CLA
```
Comment (WillsonHG) [inline contributors/contribute-to-a-gerrit-project.md]: "You will be redirected to [corporate.lfcla.com](https://corporate.lfcla.com/#/companies) to add your company to a project" Link to https://lf-docs-linux-foundation.gitbook.io/easycla/ccla-managers-and-ccla-signatories/add-a-company-to-a-project

File: contributors/contribute-to-a-gerrit-project.md (hunk 1 above)
Comment (dibyaranjanlf) [reply contributors/contribute-to-a-gerrit-project.md]: corrected!

File: contributors/contribute-to-a-gerrit-project.md (hunk 2 above)
Comment (dibyaranjanlf) [reply contributors/contribute-to-a-gerrit-project.md]: deleted the extra tab space.

File: contributors/contribute-to-a-gerrit-project.md (hunk 3 above)
Comment (dibyaranjanlf) [reply contributors/contribute-to-a-gerrit-project.md]: corrected!

File: contributors/contribute-to-a-gerrit-project.md (hunk 4 above)
Comment (dibyaranjanlf) [reply contributors/contribute-to-a-gerrit-project.md]: corrected!

File: contributors/contribute-to-a-gerrit-project.md (hunk 5 above)
Comment (dibyaranjanlf) [reply contributors/contribute-to-a-gerrit-project.md]: changed

**Pull request merged**